src/lib.rs

```diff
@@ -109,7 +109,14 @@ crate. Pretty much everything else is for ergonomics.
 #![deny(clippy)]
 #![cfg_attr(feature = "ci", deny(warnings))]
 #![cfg_attr(not(feature = "std"), no_std)]
-#![cfg_attr(not(feature = "std"), feature(core_intrinsics, extern_prelude))]
+#![cfg_attr(
+    all(not(feature = "std"), feature = "nightly"),
+    feature(core_intrinsics)
+)]
+#![cfg_attr(
```

Comment:
This is necessary as it would otherwise cause an error on nightly due to …

```rust
#![cfg_attr(feature = "ci", deny(warnings))]
```

… and feature `extern_prelude` having been stabilized with 1.30.

Reply:
You're welcome to just remove this attribute then.